addons/mail/static/src/new/attachments/attachment_viewer.xml

```diff
@@ -62,10 +63,10 @@
                     </div>
                 </div>
                 <t t-if="props.attachments.length > 1">
-                    <div class="o-mail-attachment-viewer-buttonNavigation o-mail-attachment-viewer-buttonNavigationPrevious o-mail-attachment-viewer-buttonNavigationPreviousIcon position-absolute top-0 bottom-0 start-0 align-items-center justify-content-center d-flex my-auto ms-3 rounded-circle" t-on-click="previous" title="Previous (Left-Arrow)" role="button">
+                    <div class="o-mail-attachment-viewer-buttonNavigation position-absolute top-0 bottom-0 start-0 align-items-center justify-content-center d-flex my-auto ms-3 rounded-circle" t-on-click="previous" title="Previous (Left-Arrow)" role="button">
```

Comment:
aria-label added to next but not to previous?

Reply:
Was not used. But ok, I add it too.